hardhat.config.js

```diff
@@ -86,4 +86,10 @@ for (let k in process.env) {
             }
         }
     }
+
+    if (k == "ETHERSCAN_API_KEY") {
```

Comment:
Best practise to use `===` instead of `==`.

The `tasks/verification.js` task uses the env variable `ETHERSCAN`. We should make this consistent, either here or in the task.